substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/DebugInfoBase.java

```diff
@@ -34,6 +34,7 @@
 import java.util.List;
 import java.util.Map;
 
+import com.oracle.objectfile.debuginfo.dwarf.DwarfDebugInfoBase;
```

Comment:
Please relocate imports this so it sits with the other com.oracle imports in the correct alphabetical